src/main/java/seedu/address/model/cca/CommitteeCca.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.model.cca;
```

Comment:
Is there a need to split the ccas into the different types of cca?

Reply:
Not sure if it matters but I think it doesn't really matter? If there is a real problem I can remove it and classify under CCA alone :)

Reply:
I have no big issues with it just that if it's not used then maybe can put inside a separate folder that is not in the code so that it's neater!